.packit.yaml

```diff
@@ -0,0 +1,40 @@
+# https://packit.dev/docs/configuration/
+
+specfile_path: python-dockerfile-parse.spec
+
+# to be copied to dist-git during an update
+synced_files:
+    - python-dockerfile-parse.spec
+    - .packit.yaml
+
+# name in upstream package repository/registry (e.g. in PyPI)
+upstream_project_name: dockerfile-parse
+# downstream (Fedora) RPM package name
+downstream_package_name: python-dockerfile-parse
+
+current_version_command: ["python3", "setup.py", "--version"]
+create_tarball_command: ["python3", "setup.py", "sdist", "--dist-dir", "."]
```

Comment:
it works for me even with default version and tarball commands: can we remove them?

Reply:
removed